source/snes9x/memmap.cpp

```diff
@@ -3819,6 +3819,7 @@ void CMemory::ApplyROMFixes (void)
 	//// APU timing hacks :(
 
 	Timings.APUSpeedup = 0;
+	Timings.APUAllowTimeOverflow = FALSE;
```

Comment:
Same goes for all these lines, wrap them in an #ifdef GEKKO so we don't accidentally lose them some day in a core upgrade.